modules/administration-guide/examples/snip_che-create-monitoring-namespace.adoc

```diff
@@ -0,0 +1,4 @@
+[subs="+attributes,quotes"]
+----
+$ {orch-cli} create {orch-namespace} monitoring
```

Comment:
For downstream, this should be `$ oc new-project monitoring`

Reply:
@dkwon17, see if you can click the **Approve** button in https://gitlab.cee.redhat.com/red-hat-developers-documentation/red-hat-devtools/-/merge_requests/1524.

Reply:
https://gitlab.cee.redhat.com/red-hat-developers-documentation/red-hat-devtools/-/merge_requests/1524 is merged.

Reply:
David, you're welcome to resolve this thread.

Reply:
David, you're welcome to resolve this thread.